_docs/_latest/configure/inventory/gsuite.md

```diff
@@ -52,10 +61,11 @@ You must have the **super admin** role in admin.google.com to complete these ste
 
 ## Configuring Forseti to collect G Suite data
 
-After you setup your service account above, you may need to edit [domain_super_admin_email]({% link _docs/latest/configure/inventory/index.md %}) field in 
-your `forseti_conf_server.yaml`.
+After you set up your service account above, you may need to edit the
+[domain_super_admin_email]({% link _docs/latest/configure/inventory/index.md %})
```

Comment:
...[`domain_super_admin_email`]... That's presuming the linking still works that way.

Reply:
It should from what I've seen on other pages; I'll update here.

Reply:
Resolved.